Gotcha/ScoresEndpoint.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ScoresEndpoint.swift
+//  Gotcha
+//
+//  Created by Don Miller on 5/9/18.
```

Comment:
This should be your name.

Reply:
No worries then.  I guess it really doesn't provide value since we are using git.  `Old habits`

Reply:
I find it weird that Apple does this. `NSHabits`